### DISCUSSION TITLE
CMS web builder

### DISCUSSION THREAD

**Discussion by anshmarelab85** [GrapesJS/grapesjs#6029] [Q&A]
HI @artf and Every One ,
Is There any RoadMap And Suggestion for Building Dynamic Drag and drop website builder like WordPress with CMS,
with the help of grapes JS (React JS or Next JS), Node JS, and mongodb

**Comment by bamiendev** [comment]
I have completed a similar project with MernStack and customize the ui if you need I have a paid tutorial!

**Comment by anshmarelab85** [comment]
@Aris-NBTN
first i want to see demo
please setup demo for me

**Comment by anshmarelab85** [comment]
@Aris-NBTN
any video  clip or demo docs please send me

**Comment by fredrickreuben** [comment]
I can help you get started at a small fee, grapesjs is great API and you can achieve alot if you understand how it works, here is my WhatsApp +254714707323

**Reply by fredrickreuben** [reply]
For my case I used react, nextjs, tailwindcss and adonisjs in the backend ours is an enterprise application, a landing page builder

**Reply by anshmarelab85** [reply]
first i want to see demo,
can you show me demo

**Reply by fredrickreuben** [reply]
send me a dm
